packages/AdaptiveCards/README.md

```diff
@@ -0,0 +1,20 @@
+# Microsoft.Bot.Components.Teams
+
+Actions and triggers for working with the Adaptive Cards from your BOt Framework bot. This package contains:
+
+- Actions for creating, storing, and sending Adaptive Cards
+- Triggers for responding to Adaptive Cards invokes & activities, like Action.Submit, Action.Execute
```

Comment:
nit: helpful to link to action documentation here as well. https://docs.microsoft.com/en-us/microsoftteams/platform/task-modules-and-cards/cards/cards-actions#adaptive-cards-actions